app/assets/stylesheets/components/_tooltip.scss

```diff
@@ -0,0 +1,56 @@
+//--------------------------------------------------
+// variables
+//--------------------------------------------------
+
+//--------------------------------------------------
+// mixins
+//--------------------------------------------------
+
+//--------------------------------------------------
+// classes
+//--------------------------------------------------
+.tooltip {
+  position: relative;
+    
+  &.popup {
+    background-color: $white;
+    font-family: $body-font;
+    font-size: rem-calc(12);
+    font-weight: 500;
+    line-height: 1.2;
+    padding: rem-calc(20);
+    max-width: 80%;
+
+    position: absolute;
+    right: rem-calc(52);
+    top: 0;
```

Comment:
Couple of spots here where right and top and the wrong way round

Reply:
hmm, they are alphabetically

Reply:
I think it's usually clockwise from top: https://github.com/unepwcmc/wiki/wiki/Styling